### DISCUSSION TITLE
If I modify a job, it will run immediately although it isn't scheduled for this time day.

### DISCUSSION THREAD

**Discussion by s3YwCf2ZbfJG4SHAfjQMAjtsf** [quartznet/quartznet#2036] [Q&A]
Applies to recurring jobs with a cron, in my case.

I am using the bulk DeleteJobs and ScheduleJobs methods, as there multiple jobs being modified or removed.   I can't tell if this is due to the "DeleteJobs" (I.E. some kind of cleanup) or "ScheduleJobs". 

The job is scheduled to run hourly at 48 minutes past the hour (hourly with cron).   But it is 12:36 pm.   The job is Eastern Time.  

I can't tell anything in the debugger to determine this is an "off schedule" execution.   

![image](https://github.com/quartznet/quartznet/assets/784740/4b8b3ce3-b389-4e9e-be72-71de7403acb3)

**Comment by lahma** [comment]
I'm more interested on you username, how did you come up with that? It's quite interesting I think.

**Reply by s3YwCf2ZbfJG4SHAfjQMAjtsf** [reply]
My "username" for github? :)   Create GitHub account and when entering your username, use a password generator without special characters. ;) 

But can you help me?  LOL

**Reply by lahma** [reply]
With this description of the problem, environment, libraries etc, I have no idea. LOL

**Reply by s3YwCf2ZbfJG4SHAfjQMAjtsf** [reply]
I am not sure if there is a bug in DeleteJobs that causes some kind of cleanup situation.

**Comment by s3YwCf2ZbfJG4SHAfjQMAjtsf** [comment]
When you call DeleteJobs it doesn't call UnscheduleJob(s) and it appears to trigger the job incorrectly without it.

Now I get the trigger keys: UnscheduleJobs, then call DeleteJobs.
